src/spec-node/devContainersSpecCLI.ts

```diff
@@ -830,7 +836,8 @@ async function doRunUserCommands({
 		const containerProperties = await createContainerProperties(params, container.Id, configs?.workspaceConfig.workspaceFolder, mergedConfig.remoteUser);
 		const updatedConfig = containerSubstitute(cliHost.platform, config.config.configFilePath, containerProperties.env, mergedConfig);
 		const remoteEnv = probeRemoteEnv(common, containerProperties, updatedConfig);
-		const result = await runLifecycleHooks(common, lifecycleCommandOriginMapFromMetadata(imageMetadata), containerProperties, updatedConfig, remoteEnv, stopForPersonalization);
+		const secrets = readSecretsFromFile(common);
```

Comment:
for consistency should we rename these tho `secretsP` and `remoteEnvP` (I presume from the previous usages that the 'P' stands for promise)